### ISSUE TITLE
Unable to locate class

### ISSUE THREAD

**Issue by razvanilin** [InstaPy/InstaPy#133]
I'm getting this error lately and it's almost impossible to run the script. It works if I try about 5 times to start it. Maybe Instagram changed something and it broke the script.

```
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"class name","selector":"_uikn3"}
  (Session info: chrome=57.0.2987.133)
  (Driver info: chromedriver=2.29.461571 (8a88bbe0775e2a23afda0ceaf2ef7ee74e822cc5),platform=Linux 4.4.0-72-generic x86_64)
```

**Comment by sugionodjojo** [comment]
i think this belong to like-util.py

the clone git give you updated like_util.py, ( where something like  "_uikn3" mentioned)

try to upload like_util.py, reboot the droplet, pip install ., and then python3.5 quickstart.py again

**Comment by razvanilin** [comment]
@sugionodjojo I don't really understand what you mean, but if you're trying to say that I should update the like_util.py, then I can assure you that I pulled the latest changes from the master branch and I still encounter that issue.

**Comment by sugionodjojo** [comment]
nope, sorry, mistaken.... I also experience once, and the line above that could really help, that explain what went wrong. something to do with /instapy/login_util.py and like_util.py

i also always reboot the droplet, start with pip install .
and then python3.5 quickstart.py

then i have successful login......

**Comment by GaryYiu** [comment]
@razvanilin Can you provide more information about the error? Judging from the error message you provided, it might be related to fix #132 . Instagram changed some of its account query language to Graphql that cause this. But there is no official changes committed to the master. You have to fix it manually according to the commit in #132 . I did the same and my script is running fine.

**Comment by sugionodjojo** [comment]
@GaryYiu  I did fix #132 but still got the same message as @razvanilin 

furthermore fix #132 is like_util.py, but according to the result , it related with login_util.py

root@ubuntu-1gb-nyc2-01:/InstaPy# python3.5 quickstart.py
Traceback (most recent call last):
  File "quickstart.py", line 15, in <module>
    InstaPy(username='****', password='********')\
  File "/InstaPy/instapy/instapy.py", line 73, in login
    if not login_user(self.browser, self.username, self.password):
  File "/InstaPy/instapy/login_util.py", line 18, in login_user
    login_div = browser.find_element_by_class_name('_uikn3')
  File "/usr/local/lib/python3.5/dist-packages/selenium/webdriver/remote/webdriver.py", line 413, in find_element_by_class_name
    return self.find_element(by=By.CLASS_NAME, value=name)
  File "/usr/local/lib/python3.5/dist-packages/selenium/webdriver/remote/webdriver.py", line 752, in find_element
    'value': value})['value']
  File "/usr/local/lib/python3.5/dist-packages/selenium/webdriver/remote/webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.5/dist-packages/selenium/webdriver/remote/errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"class name","selector":"_uikn3"}
  (Session info: chrome=57.0.2987.133)
  (Driver info: chromedriver=2.29.461571 (8a88bbe0775e2a23afda0ceaf2ef7ee74e822cc5),platform=Linux 4.4.0-72-generic x86_64)


i think related with business account not using graphql , @vlntdds mentions earlier on fix #132

**Comment by sugionodjojo** [comment]
@GaryYiu @razvanilin 
for sure it is reboot problem
just reboot, pip install . and python3.5 quickstart.py

it works for me, i mean login successful....

**Comment by eduardocalazansjr** [comment]
The login page now changes all the classes, maybe it's time related.

As i said, instagram is changing a lot of things, time to wait and refactor instapy

**Comment by timgrossmann** [comment]
solved by #145
